WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -859,7 +868,71 @@ public boolean isNewPost() {
+                “Post title” remains a draft. Remember to publish it!
+            1 month
+                Don’t leave it hanging! “Post title” is waiting to be published.
+            * */
+
+        AlarmManager alarmManager = (AlarmManager) context.getSystemService(Context.ALARM_SERVICE);
+
+        Intent intent = getNotificationsPendingDraftReceiverIntent(context, localPostId);
+
+        PendingIntent alarmIntentOneDay = getOneDayAlarmIntent(context, intent);
+        PendingIntent alarmIntentOneWeek = getOneWeekAlarmIntent(context, intent);;
+        PendingIntent alarmIntentOneMonth = getOneMonthAlarmIntent(context, intent);;
+
+        long now = System.currentTimeMillis();
+
+        alarmManager.set(AlarmManager.RTC_WAKEUP, now + NotificationsPendingDraftsReceiver.ONE_DAY, alarmIntentOneDay);
+        alarmManager.set(AlarmManager.RTC_WAKEUP, now + NotificationsPendingDraftsReceiver.ONE_WEEK, alarmIntentOneWeek);
+        alarmManager.set(AlarmManager.RTC_WAKEUP, now + NotificationsPendingDraftsReceiver.ONE_MONTH, alarmIntentOneMonth);
+    }
+
+    private void cancelPendingDraftAlarms(Context context, long localPostId) {
+        AlarmManager alarmManager = (AlarmManager) context.getSystemService(Context.ALARM_SERVICE);
+
+        Intent intent = getNotificationsPendingDraftReceiverIntent(context, localPostId);
+
+        PendingIntent alarmIntentOneDay = getOneDayAlarmIntent(context, intent);
+        PendingIntent alarmIntentOneWeek = getOneWeekAlarmIntent(context, intent);;
+        PendingIntent alarmIntentOneMonth = getOneMonthAlarmIntent(context, intent);;
+
+        alarmManager.cancel(alarmIntentOneDay);
+        alarmManager.cancel(alarmIntentOneWeek);
+        alarmManager.cancel(alarmIntentOneMonth);
+    }
+
+    private Intent getNotificationsPendingDraftReceiverIntent(Context context, long localPostId) {
+        Intent intent = new Intent(context, NotificationsPendingDraftsReceiver.class);
+        intent.putExtra(POST_ID_EXTRA, localPostId);
+        return intent;
+    }
+    private PendingIntent getOneDayAlarmIntent(Context context, Intent notifPendingDraftReceiverIntent) {
+        PendingIntent alarmIntentOneDay = PendingIntent.getBroadcast(context, BROADCAST_BASE_REQUEST_CODE, notifPendingDraftReceiverIntent, PendingIntent.FLAG_UPDATE_CURRENT);
+        return alarmIntentOneDay;
+    }
+
+    private PendingIntent getOneWeekAlarmIntent(Context context, Intent notifPendingDraftReceiverIntent) {
+        PendingIntent alarmIntentOneWeek = PendingIntent.getBroadcast(context, BROADCAST_BASE_REQUEST_CODE + 1, notifPendingDraftReceiverIntent, PendingIntent.FLAG_UPDATE_CURRENT);
+        return alarmIntentOneWeek;
+    }
+
+    private PendingIntent getOneMonthAlarmIntent(Context context, Intent notifPendingDraftReceiverIntent) {
+        PendingIntent alarmIntentOneMonth = PendingIntent.getBroadcast(context, BROADCAST_BASE_REQUEST_CODE + 1, notifPendingDraftReceiverIntent, PendingIntent.FLAG_UPDATE_CURRENT);
```

Comment:
`BROADCAST_BASE_REQUEST_CODE + 1` it's the same value of the `Week` branch. Is it ok?